third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -353,6 +353,20 @@ func resourceDataprocCluster() *schema.Resource {
 								},
 							},
 						},
+						"endpoint_config": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"enable_http_port_access": {
+										Type:     schema.TypeBool,
+										Optional: true,
+										Default:  false,
```

Comment:
This is totally fine to include, but also not necessary- it'll be false-by-default, by default